test/acceptance_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule AcceptanceTest do
+  use ExUnit.Case
+
+  describe "acceptance" do
+    test "the game works when run from the command line" do
+      Mix.Shell.Process.cmd("mix escript.build")
```

Comment:
**suggestion** Consider removing this cmd, and updating `mix.exs` to call `escript.build` when you run your tests.